autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/LoadAlertConfigurationRequest.java

```diff
@@ -11,12 +14,19 @@
 public class LoadAlertConfigurationRequest implements Json {
 
     @ApiModelProperty(LoadAlertJsonProperties.LOAD_ALERT_CONFIGURATION_MIN_RESOUCE_VALUE)
+    @Min(value = 1)
+    @Digits(fraction = 0, integer = 3)
     private @Valid Integer minResourceValue;
 
     @ApiModelProperty(LoadAlertJsonProperties.LOAD_ALERT_CONFIGURATION_MAX_RESOUCE_VALUE)
+    @Max(value = 200)
+    @Digits(fraction = 0, integer = 3)
     private @Valid Integer maxResourceValue;
 
     @ApiModelProperty(LoadAlertJsonProperties.LOAD_ALERT_CONFIGURATION_COOL_DOWN_MINS_VALUE)
+    @Min(value = 5)
```

Comment:
Think the 'min' should still be about 2 minutes here, to allow for enhancements in scale-up performance.
Also scale-down can finish within 2 minutes, or even faster.

Reply:
ok